docs/source/content/overview/linear_SVM.rst

```diff
@@ -2,67 +2,157 @@
 Linear Support Vector Machines
 ==========================
 
-A **Support Vector Machine** (SVM for short) is another machine learning algorithm that is used to classify data. The point of SVM's are to try and find a line or hyperplane to divide a dimensional space which best classifies the data points. If we were trying to divide two classes A and B we would try best separate the two classes with a line. So on one side of the line/hyperplane would be data from class A and on the otherside would be from class B. This alogorithm is very useful in classifying because we just have to caluclate the best line or hyperplane once and any knew data points we can easily classify just by seeing which side of the line it falls on. Unlike KNN, where we would have to calculate each data points nearest neighbors. 
+A **Support Vector Machine** (SVM for short) is another machine learning algorithm that is used to classify data.
+The point of SVM's are to try and find a line or **hyperplane** to divide a dimensional space which best classifies
+the data points. If we were trying to divide two classes A and B we would try best separate the two classes with a 
+line. So on one side of the line/hyperplane would be data from class A and on the otherside would be from class B. 
+This alogorithm is very useful in classifying because we just have to caluclate the best line or hyperplane once 
+and any new data points we can easily classify just by seeing which side of the line it falls on. Unlike KNN, where 
+we would have to calculate each data points nearest neighbors. 
+
+Hyperplane
+----------
+A **hyperplane** depends on the space it is in, but it divides the space into two disconnected parts. So 
+1-dimensional space this would just be a point, 2-d space a line, 3-d space a plane, and so on. 
 
 How do we find the best hyperplane/line?
 ----------------------------------------
 
-You might be wondering that there could be multiple lines that split the data well. As you can see in the graph below every line splits the squares and the circles, so which one do we choose?
+You might be wondering that there could be multiple lines that split the data well. In fact there is a infinite
+amount of lines that can divide two classes.  As you can see in the graph below every line splits the squares and
```

Comment:
Should be one space between sentences,
As you can see in the graph below`,` every